TeamAikon/oreid
# [ENG-1143] fix footer logo

How to test:
run the site with `yarn dev` then resize the window to see if the logo in the footer overlaps any elements.

Screenshot showing the bug is resolved:
<img width="1049" alt="Captura de Tela 2022-06-16 às 21 34 32" src="https://user-images.githubusercontent.com/1014611/174200098-cf07f473-dd2b-4078-b53f-12e96aad5eef.png">
